docs/xks/developer-guide/api-migrations.md

```diff
@@ -9,6 +9,82 @@ resources if you are utilizing them, but we will of course inform you when it's
 
 This page is about sharing that information.
 
+# Kubernetes 1.25
+
+## CronJob
+
+Moving CronJob apiVersion from `batch/v1beta1` to `batch/v1`
+The only thing you need to do is to change the API version.
+
+From:
+
+````yaml
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: foo
+````
+
+To:
+
+````yaml
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: foo
+````
+## HorizontalPodAutoscaler
+
+Moving HorizontalPodAutoscaler apiVersion from `autoscaling/v2beta1` to `autoscaling/v2`
+The only thing you need to do is to change the API version.
+
+From:
+
+````yaml
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: foo
+````
+
+To:
+
+````yaml
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: foo
+````
+## PodDisruptionBudget
+
+Moving HorizontalPodAutoscaler apiVersion from `autoscaling/v2beta1` to `autoscaling/v2`
+The only thing you need to do is to change the API version.
+
+From:
+
+````yaml
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: foo
+````
+
+To:
+
+````yaml
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: foo
+````
+Other noticeable changes is that in `policy/v1` an empty `spec.selector {}` selects all pods in the namespace, when the previous `policy/v1beta1` did not select any pods.
+
+## PodSecurityPolicy
+
+PodSecurityPolicy in the policy/v1beta1 API version will no longer be served in v1.25, and the PodSecurityPolicy admission controller will be removed. More informaiton can be found [here.](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#psp-v125)
```

Comment:
Something about that we are not using PSP in general and the users don't need to think about it.